kclvm/loader/src/snapshots/kclvm_loader__tests__builtin_call_1.snap

```diff
@@ -383,6 +369,20 @@ expression: "format!(\"{:#?}\", p.symbols.values())"
                 },
                 kind: Value,
             },
+             SymbolRef {
```

Comment:
The CI failed according to here.

Reply:
The error was arising due to the indentation of `SymbolRef`, fixed it.